<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OutgoingMailSettingsWidget</class>
 <widget class="QWidget" name="OutgoingMailSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>341</width>
    <height>360</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="widgetLayout" stretch="0,0,0,1">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="headerWidget" native="true">
     <layout class="QVBoxLayout" name="headerLayout">
      <item>
       <layout class="QHBoxLayout" name="serviceTypeDropdownLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="serviceTypeDropdown">
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="serviceTypeDropdownSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="statusPage">
         <layout class="QVBoxLayout" name="statusPageLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="configurationStatusLabel">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="configurationStatusHintLabel"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="progressIndicatorPage">
         <layout class="QHBoxLayout" name="progressIndicatorPageLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="nx::vms::client::desktop::BusyIndicatorWidget" name="progressIndicator" native="true"/>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>296</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="formLayout">
     <item>
      <widget class="QGroupBox" name="connectionSettingsGroupBox">
       <property name="title">
        <string>Connection Settings</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <layout class="QGridLayout" name="connectionSettingsLayout" columnstretch="2,5,2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="2" column="0">
         <widget class="QLabel" name="userLabel">
          <property name="text">
           <string>User</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="protocolLabel">
          <property name="text">
           <string>Security Protocol</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <spacer name="connectionSettingsHorizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1">
         <widget class="nx::vms::client::desktop::InputField" name="emailInput" native="true">
          <property name="focusPolicy">
           <enum>Qt::StrongFocus</enum>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="nx::vms::client::desktop::InputField" name="serverAddressInput" native="true">
          <property name="focusPolicy">
           <enum>Qt::StrongFocus</enum>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="passwordLabel">
          <property name="text">
           <string>Password</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="serverAddressLabel">
          <property name="text">
           <string>Server Address</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="nx::vms::client::desktop::PasswordInputField" name="passwordInput" native="true">
          <property name="focusPolicy">
           <enum>Qt::StrongFocus</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QnWordWrappedLabel" name="emailInputHint" native="true"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="emailLabel">
          <property name="text">
           <string>Email</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="nx::vms::client::desktop::InputField" name="userInput" native="true">
          <property name="focusPolicy">
           <enum>Qt::StrongFocus</enum>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QComboBox" name="protocolComboBox"/>
          </item>
          <item>
           <spacer name="securityProtocolComboBoxSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="4" column="1">
         <widget class="QnWordWrappedLabel" name="passwordInputHint" native="true"/>
        </item>
        <item row="6" column="1">
         <widget class="QnWordWrappedLabel" name="serverAddressInputHint" native="true"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="emailSettingsGroupBox">
       <property name="title">
        <string>Email Settings</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <layout class="QGridLayout" name="emailSettingsLayout" columnstretch="2,5,2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="1" column="0">
         <widget class="QLabel" name="supportSignatureLabel">
          <property name="text">
           <string>Support Signature</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="systemSingnatureLabel">
          <property name="text">
           <string>System Signature</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <spacer name="emailSettingsHorizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1">
         <widget class="nx::vms::client::desktop::InputField" name="systemSignatureInput" native="true">
          <property name="focusPolicy">
           <enum>Qt::StrongFocus</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="nx::vms::client::desktop::InputField" name="supportSignatureInput" native="true">
          <property name="focusPolicy">
           <enum>Qt::StrongFocus</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>nx::vms::client::desktop::InputField</class>
   <extends>QWidget</extends>
   <header location="global">nx/vms/client/desktop/common/widgets/input_field.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>nx::vms::client::desktop::PasswordInputField</class>
   <extends>QWidget</extends>
   <header location="global">nx/vms/client/desktop/common/widgets/password_input_field.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>nx::vms::client::desktop::BusyIndicatorWidget</class>
   <extends>QWidget</extends>
   <header location="global">nx/vms/client/desktop/common/widgets/busy_indicator.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QnWordWrappedLabel</class>
   <extends>QWidget</extends>
   <header location="global">ui/widgets/word_wrapped_label.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>emailInput</tabstop>
  <tabstop>userInput</tabstop>
  <tabstop>passwordInput</tabstop>
  <tabstop>serverAddressInput</tabstop>
  <tabstop>protocolComboBox</tabstop>
  <tabstop>systemSignatureInput</tabstop>
  <tabstop>supportSignatureInput</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
